src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -416,6 +393,49 @@ class StandardReader : public FitsColumnReader {
     Key<T> _key;
 };
 
+class AngleReader : public FitsColumnReader {
+public:
+
+    static std::unique_ptr<FitsColumnReader> make(
+        Schema & schema,
+        FitsSchemaItem const & item,
+        FieldBase<afw::geom::Angle> const & base=FieldBase<afw::geom::Angle>()
+    ) {
+        return std::unique_ptr<FitsColumnReader>(new AngleReader(schema, item, base));
+    }
+
+    AngleReader(
+        Schema & schema,
+        FitsSchemaItem const & item,
+        FieldBase<afw::geom::Angle> const & base
+    ) :
+        _column(item.column),
+        _key(schema.addField<afw::geom::Angle>(item.ttype, item.doc, "", base))
+    {
+        // We require an LSST-specific key in the headers before parsing a column
+        // as Angle at all, so we don't need to worry about other units or other
+        // spellings of radians.  We do continue to support no units for backwards
+        // compatibility.
+        if (!item.tunit.empty() && item.tunit != "rad") {
+            throw LSST_EXCEPT(afw::fits::FitsError, "Angle fields must be persisted in radians.");
```

Comment:
I guess supporting us reading in non-radian fields (e.g. from other data sets) is work for an other time?